code/Model/Submission/SubmittedFileField.php

```diff
@@ -41,25 +41,16 @@ public function getFormattedValue()
     {
         $name = $this->getFileName();
         $link = $this->getLink(false);
-        $title = _t(__CLASS__ . '.DOWNLOADFILE', 'Download File');
-        $message = _t(__CLASS__ . '.INSUFFICIENTRIGHTS', 'You don\'t have the right permissions to download this file');
-        $file = $this->getUploadedFileFromDraft();
-
         if ($link) {
-            if ($file->canView()) {
-                return DBField::create_field('HTMLText', sprintf(
-                    '%s - <a href="%s" target="_blank">%s</a>',
-                    htmlspecialchars($name, ENT_QUOTES),
-                    htmlspecialchars($link, ENT_QUOTES),
-                    htmlspecialchars($title, ENT_QUOTES)
-                ));
-            } else {
-                return DBField::create_field('HTMLText', sprintf(
-                    '<i class="icon font-icon-lock"></i> %s - <em>%s</em>',
-                    htmlspecialchars($name, ENT_QUOTES),
-                    htmlspecialchars($message, ENT_QUOTES)
-                ));
-            }
+            $title = _t(__CLASS__ . '.DOWNLOADFILE', 'Download File');
+            $message = _t(__CLASS__ . '.YOUMUSTBELOGGEDIN', 'You must be logged in to view this file');
+            return DBField::create_field('HTMLText', sprintf(
+                '<i class="icon font-icon-lock"></i> %s - <a href="%s" target="_blank">%s</a> - <em>%s</em>',
+                htmlspecialchars($name, ENT_QUOTES),
+                htmlspecialchars($link, ENT_QUOTES),
+                htmlspecialchars($title, ENT_QUOTES),
+                htmlspecialchars($message, ENT_QUOTES)
+            ));
```

Comment:
```suggestion
            $title = _t(__CLASS__ . '.DOWNLOADFILE', 'Download File');
            return DBField::create_field('HTMLText', sprintf(
                '<i class="icon font-icon-lock"></i> %s - <a href="%s" target="_blank">%s</a></em>',
                htmlspecialchars($name, ENT_QUOTES),
                htmlspecialchars($link, ENT_QUOTES),
                htmlspecialchars($title, ENT_QUOTES),
            ));
```

Reply:
Since we return previous behaviour, message is not longer required and it's a little bit confusing to see this message when you have already logged in.  
![screencapture-localhost-90-sink413new-admin-pages-edit-EditForm-15-field-Submissions-item-1-edit-2023-09-05-15_59_13](https://github.com/silverstripe/silverstripe-userforms/assets/87288324/cf24a9a8-4dda-4cb5-bafe-fdff84e63343)

Reply:
One of the AC's in https://github.com/silverstripe/silverstripe-userforms/issues/1225 is "If the file is protected, the email tells you that you have to be logged in to see the file."

When you're receiving the email you might not be logged in, and the email has no way of knowing if you are logged int.  So the email should include a reminder that you need to be logged so that not-logged-in CMS users don't get confused when they cannot view the file.

Reply:
For me it's completely misleading message. Before recipient didn't get any additional messages in email, so I think this AC is redundant.

Reply:
Updated